src/pages/nav/Nav.jsx

```diff
@@ -6,17 +6,21 @@ import { useAuth } from "../../context/authContext";
 export default function Nav() {
   const { token, logoutHandler } = useAuth();
 
+  const handleClick = () => {
+    logoutHandler();
+  };
+
   return (
     <nav className="nav">
       <div className="nav-title">
         <p>
-          <a href="/"> Glee </a>
+          <Link to="/"> Glee </Link>
           <span className="text-sm">v1.0</span>
         </p>
       </div>
       <div className="nav-pills">
         {token ? (
-          <a onClick={logoutHandler} className=" btn-filled">
+          <a onClick={handleClick} className=" btn-filled">
```

Comment:
Can use `<Link>` here for consistency in code.

Reply:
The link was giving an error because I didn't need the `to` attribute in it.